tests/next_tests/definitions.py

```diff
@@ -130,13 +129,18 @@ class ProgramFormatterId(_PythonObjectIdMixin, str, enum.Enum):
 COMMON_SKIP_TEST_LIST = [
     (REQUIRES_ATLAS, XFAIL, BINDINGS_UNSUPPORTED_MESSAGE),
     (USES_APPLIED_SHIFTS, XFAIL, UNSUPPORTED_MESSAGE),
-    (USES_IF_STMTS, XFAIL, UNSUPPORTED_MESSAGE),
     (USES_NEGATIVE_MODULO, XFAIL, UNSUPPORTED_MESSAGE),
     (USES_REDUCTION_WITH_ONLY_SPARSE_FIELDS, XFAIL, REDUCTION_WITH_ONLY_SPARSE_FIELDS_MESSAGE),
-    (USES_SCAN_IN_FIELD_OPERATOR, XFAIL, UNSUPPORTED_MESSAGE),
     (USES_SPARSE_FIELDS_AS_OUTPUT, XFAIL, UNSUPPORTED_MESSAGE),
 ]
+# Markers to skip because of missing features in the domain inference
+DOMAIN_INFERENCE_SKIP_LIST = [
+    (USES_DYNAMIC_OFFSETS, XFAIL, UNSUPPORTED_MESSAGE),
+    (USES_STRIDED_NEIGHBOR_OFFSET, XFAIL, UNSUPPORTED_MESSAGE),
+]
 DACE_SKIP_TEST_LIST = COMMON_SKIP_TEST_LIST + [
+    (USES_IF_STMTS, XFAIL, UNSUPPORTED_MESSAGE),
```

Comment:
Sorry, another question. I can clean it up in a seprate PR, if needed.

Isn't `USES_IF_STMTS` a subset of `STARTS_FROM_GTIR_PROGRAM`, which is already in the list? Same for `USES_SCAN_IN_FIELD_OPERATOR`?

Reply:
`USES_IR_IF_STMTS` is conceptually a subset of `STARTS_FROM_GTIR_PROGRAM`, but not in practice. I just never noticed. #1691 switches to exclusively using `itir.Program` so I will remove the marker completely there.